lib/screens/scan_screen.dart

```diff
@@ -0,0 +1,166 @@
+import 'dart:convert';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:scan/scan.dart';
+import 'package:http/http.dart' as http;
+import 'package:image_picker/image_picker.dart';
+
+// ignore: must_be_immutable
+class ScanScreen extends StatefulWidget {
+  dynamic data;
```

Comment:
**question:** can you expand a little bit why you made a stateful widget mutable?

Reply:
All fields in a class extending StatelessWidget should be final.
At that time to ignore the warning, I put "ignore: must_be_immutable". But forgot to change it latter

But what I should have done was make class variable final, (final dynamic data;)
and make it's constructor const,
const ScanScreen({Key? key, required this.data}) : super(key: key);